### PR TITLE
Added server side error catching examples to docs

### PR DESCRIPTION
**Pull request by fawzisf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,26 @@ window.onerror = (msg, url, line) => {
...
+```jsx
+process.on('uncaughtException', function (err) {
+  log.error("Server Crashed!", err);
+  console.error(err.stack);
+  process.exit(7);
+};
+```
+### Catch-all Meteor's errors example: [*Server*]
+```jsx
+//store original Meteor error
```
Comment (dr-dimitru) [inline README.md]: add one space after `//`

File: README.md [hunk 2]
```diff
@@ -67,6 +67,26 @@ window.onerror = (msg, url, line) => {
...
+let originalMeteorDebug = Meteor._debug;
```
Comment (dr-dimitru) [inline README.md]: We can safely use `const` here

File: README.md [hunk 3]
```diff
@@ -67,6 +67,26 @@ window.onerror = (msg, url, line) => {
...
+Meteor._debug =(message, stack) => {
+  let error = new Error(message);
```
Comment (dr-dimitru) [inline README.md]: `const` will work here too

**Review by dr-dimitru — CHANGES_REQUESTED** [review]

**Comment by dr-dimitru** [comment]
Everything looks great to me, I've requested minor changes.

Will you be able add same code snippets to?:
 - [File](https://atmospherejs.com/ostrio/loggerfile);
 - [Mongo](https://atmospherejs.com/ostrio/loggermongo);
 - [Console](https://atmospherejs.com/ostrio/loggerconsole).

**Comment by fawzisf** [comment]
@dr-dimitru sure !

**Review by dr-dimitru — APPROVED** [review]

**Comment by dr-dimitru** [comment]
@fawzisf thanks a lot, merged.

**Pull request merged**